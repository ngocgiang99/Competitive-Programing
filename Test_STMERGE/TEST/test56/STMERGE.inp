9
10 9
557167313 307598383 742259617 320930088 892961004 38816721 297308746 236874098 529509977 
473172891 461318968 393573535 953697036 793684500 233749631 585220437 883749691 715255296 
722924660 982198393 175508316 703179877 483724716 675052335 545832994 818506374 957482749 
633179869 309629571 929111018 323446140 891201319 60569524 57771092 654190204 289029619 
638289706 691602891 478551956 712300390 72733337 550685331 853876377 853014500 427680760 
668113713 764655519 252590612 694703482 304850538 500453627 379231960 317385110 390712132 
187626819 835984728 921075967 665358594 971040587 531971830 675060395 791344939 324069416 
320346 327300559 70939011 788095792 764846183 381343200 662006799 38632951 328579651 
847664275 300367636 174017619 983199059 256050653 449569047 296491608 992807483 480718834 
455507677 863723099 657356769 35582793 667508076 670640370 624773993 325904021 991899655 
9 7
366659276 267961397 417363822 62377462 801890083 220551098 443361213 
539878020 65619727 290273947 711855987 357604873 642324037 326313757 
562732646 438163690 754393786 278167494 12706986 397109707 47963299 
230683993 515391125 579046290 954188405 277679325 772721339 611231076 
204516068 180272533 40900593 830735013 316882150 614749448 583204685 
306773755 757400497 271684858 890710051 331816010 177875640 905935217 
481508872 772954808 354670001 351109296 543880402 722069162 258472081 
119246141 286395542 193439428 188940685 492982750 821052839 381629420 
156190490 389674903 252617884 639499079 735102956 760763821 81495626 
2 2
852576165 973525957 
419586664 94772599 
4 8
464834489 503115993 464043663 590919094 780246608 717062358 971382220 192273303 
734621331 265985889 846221182 672764442 812051751 625165488 92134508 431019411 
101779184 392030008 270064026 623441559 725219604 987582381 329637965 482698235 
117364471 127974150 490729482 248890069 244992063 427083894 33074326 25824378 
3 6
413412855 23439997 640527569 310997939 580153968 873473718 
852704933 424642345 852179592 835755831 928178418 555958283 
196555523 475797079 713470620 373101262 961305318 738582698 
9 5
609811318 55835910 90038494 737619723 658546232 
252599935 170720383 785107119 848718577 985351275 
667564946 105102983 497990567 621648990 952600635 
293734474 665219230 185213485 632994529 972841451 
442868315 923792877 34872358 138412960 957043107 
191479796 379828978 329810910 280724859 971200727 
525859432 871319386 941725992 654462826 834542574 
795864329 405983791 455242434 777164584 196169760 
583964758 116483125 172327932 812071444 279411617 
7 6
713565337 854423124 809053693 901585221 877767078 557946614 
439801405 351414468 154187630 563044939 291669611 658416 
237237863 526636377 132721436 142691823 220169031 865854307 
332228902 398164518 810397441 223425520 454483870 175168570 
487470794 740107459 921387479 260725775 932021273 683802595 
906088809 180645835 537707170 976464788 451032102 11610578 
19263066 43765339 100004901 297148618 676216181 56954109 
4 2
698041449 944544896 
704577403 719088480 
139949036 648798247 
539415100 147570568 
7 7
763370258 251069365 16330926 689103930 380763475 352167883 555999896 
30475887 797903508 658121200 888624228 680220765 698002088 76601480 
540923398 233267849 810569638 684604473 636987317 182551480 391908835 
461335528 30299108 615624944 38798322 814772922 320858442 715225460 
810131627 172671507 443610 470160075 372786464 640888216 844641738 
684089017 965099688 37991216 426487695 253056835 834243288 692645118 
540936158 965493934 168661379 547877223 613076652 186433408 8178369 
