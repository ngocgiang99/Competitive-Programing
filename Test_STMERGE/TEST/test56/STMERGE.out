209920758
156190490
419586664
117364471
196555523
228163842
19263066
287519604
221205736
